### ISSUE TITLE
WebEx Won't Work with Virtual Camera

### ISSUE THREAD

**Issue by One-JK** [obsproject/obs-studio#7751]
### Operating System Info

Windows 10

### OBS Studio Version

28.1.2

### OBS Studio Log URL

No log

### OBS Studio Crash Log URL

No crash

### Expected Behavior

OBS virtual camera would continue to work properly with WebEx

### Current Behavior

After the 28.1.2 update my WebEx ceased to be able to use the virtual web cam.  It works fine for Teams.  I reverted to 28.0.1 based on advice from another user and everything works again.

WebEx will display a black screen and then after some seconds it will say that it cannot find the camera and I need to switch to another.

### Steps to Reproduce

1. Start a webex
2. Start Camera
3. Error occurs
...

### Anything else we should know?

If you want me to upgrade again and reproduce the issue and capture a log, just let me know.

**Comment by Fenrirthviti** [comment]
We currently suspect this is a bug in WebEx that they will need to fix, but are still investigating.

**Comment by valtorance** [comment]
Having the same issue; was working before 28.1 - do you recommend downgrade?

**Comment by Hunt-David** [comment]
Confirming downgrade to 28.0.3 worked for me as a temporary workaround

**Comment by RytoEX** [comment]
We've reverted a commit that was reportedly breaking Webex, which may have also affected GoToMeeting. Please retest in a future release of OBS. If it is not fixed in the next release, please feel free to open a new Issue with an OBS log and Steps To Reproduce.

**Comment by Hunt-David** [comment]
Thanks Ryan!

**Comment by One-JK** [comment]
Great work Ryan!!  Much appreciated.Sent using the mobile mail appOn 11/22/22 at 3:28 PM, Ryan Foster wrote:
                
            
            
                
                    From: "Ryan Foster" ***@***.***>Date: November 22, 2022To: "obsproject/obs-studio" ***@***.***>Cc: "One-JK" ***@***.***>,"Author" ***@***.***>Subject: Re: [obsproject/obs-studio] WebEx Won't Work with Virtual Camera (Issue #7751)
                
                
We've reverted a commit that was reportedly breaking Webex, which may have also affected GoToMeeting. Please retest in a future release of OBS. If it is not fixed in the next release, please feel free to open a new Issue with an OBS log and Steps To Reproduce.

**Comment by llvs** [comment]
While I had no issues with 29.0 in WebEx using the virtual camera, it seems now again broken with 29.1.1: WebEx complains that it can not connect to the camera. Can anybody confirm my observation?

**Comment by RytoEX** [comment]
> While I had no issues with 29.0 in WebEx using the virtual camera, it seems now again broken with 29.1.1: WebEx complains that it can not connect to the camera. Can anybody confirm my observation?

Works fine for me with OBS Studio 29.1.1 in Cisco Webex Meetings Web App and in the local Cisco Webex Meetings on Windows 11. Please provide more specific information on how exactly you tried to connect OBS Virtual Camera to CIsco Webex. Please also consider using our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) to iron out the details.

**Comment by llvs** [comment]
Thank you for your response. Maybe it had to do with the following: It seemed like I still had obs-virtualcam 2.05 installed which WebEx might have tried to use. This would be weird as I had already used WebEx with OBS 29 successfully and WebEx usually uses the configuration it had last time. OBS 29 is incompatible with obs-virtualcam and brings the native virtual cam with it. I uninstalled obs-virtualcam and I could now set the native virtual cam in the WebEx settings. It looks like this fixed the issue. Let's hope that it stays like this. I am running Win10 21H2 btw.
